recipe/meta.yaml

```diff
@@ -32,7 +31,7 @@ requirements:
     - pybind11
     - protobuf
     - libprotobuf
-    - pytest-runner 4.4
+    - pytest-runner 6.0.0
     - numpy   1.19  # [py<310]
```

Comment:
I think we handling those numpy version in cbc yaml already

Reply:
No, it's not, see https://github.com/AnacondaRecipes/aggregate/blob/73fe6d1efed6d5a409beeef0c7df9750d5bf7a80/conda_build_config.yaml#L153-L163

Reply:
you mean this is different:
 python 3.8-3.9
  - 1.16    # [not (osx and arm64)]
  - 1.16    # [not (osx and arm64)]
  # python 3.8-3.9
  - 1.19    # [osx and arm64]
  - 1.19    # [osx and arm64]
? 
if so, yes go ahead